src/render.py

```diff
@@ -1,11 +1,77 @@
-class RenderingManager():
-    def __init__(self, gameState, networkingManager, databaseManager):
+import tkinter as tk
+from tkinter import *
+from tkinter import ttk
+from PIL import ImageTk, Image
```

Comment:
Is there a reason you are importing PIL when its not being used? Same with some of the unused tkinter imports

Reply:
this was partially addressed via discord. PIL was added for potential usage later. That is fine but I would like to see it added to requirements.txt then.